test/models/content_view_version_import_history_test.rb

```diff
@@ -0,0 +1,41 @@
+require 'katello_test_helper'
+
+module Katello
+  class ContentViewVersionImportHistoryTest < ActiveSupport::TestCase
+    def setup
+      User.current = User.find(users(:admin).id)
+      @cvv = create(:katello_content_view_version, :major => 1, :minor => 0)
+      ::Katello::ContentViewVersionImportHistory.destroy_all
```

Comment:
Why do we have to clear all the records?

Reply:
Prolly not necessary. Just added it because we use `ContentViewVersionImportHistory.last` in a few places